scripts/tools/tools-object.js

```diff
@@ -2,7 +2,7 @@ const schema = require("./tools-schema.json");
 const axios = require('axios')
 const Ajv = require("ajv")
 const addFormats = require("ajv-formats")
-const Fuse = require("fuse.js")
+import Fuse from 'fuse.js'
```

Comment:
Remove these changes from the PR. It will break during execution in Github Actions.